DP_Project1/DP_Project1/Program.cs

```diff
@@ -0,0 +1,174 @@
+﻿/*using System;
+
+namespace DP_Project1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
+        }
+    }
+}*/
+
+
+
+using System;
+using System.Collections.Generic;
+namespace DP_Project1
+{
+
+
+    public interface PizzaBuilder
+    {
+        void Reset();
+        void BuildDough();
+        void BuildSause();
+        void BuildTopping();
+    }
+
+
+    public class WhiteBuilder : PizzaBuilder
+    {
+        private Product product = new Product();
+
+        public WhiteBuilder() { }
+       public  void Reset() {
+            this.product = new Product();
+        }
+       public  void BuildDough() {
+            this.product.Add("BuildDough");
+        }
+       public  void BuildSause() {
```

Comment:
It would be nice if you specified some sauces like (White sauce, red sauce) or what you want rather than just writing BuildSause.